### PR TITLE
google font https and content img width 100%

### PR DESCRIPTION
**Pull request by dimitrieh**

### CODE REVIEW

File: theme-white/style.css [hunk 1]
```diff
@@ -7,7 +7,7 @@ Instead, edit the stylus (.styl) files and compile it to CSS on your machine.
 /* ----------------------------------------------------------------------------
  * Fonts
  */
-@import url("http://fonts.googleapis.com/css?family=Montserrat:700|Open+Sans:300");
+@import url("https://fonts.googleapis.com/css?family=Montserrat:700|Open+Sans:300");
```
Comment (malthe) [inline theme-white/style.css:10]: Alternatively, just use `//` instead of `https://`.

Comment (rstacruz) [inline theme-white/style.css:10]: hmm? — I thought I fixed this, let me check.

Comment (rstacruz) [inline theme-white/style.css:10]: ahh — indeed it's been fixed in the latest version. it now uses `//`:
https://github.com/rstacruz/flatdoc/blob/gh-pages/theme-white/style.css

**Comment by rstacruz** [comment]
Thanks! this was fixed in a recent version.

**Pull request closed without merging**